configurations/product/mayastor_config.yaml

```diff
@@ -139,6 +139,7 @@ product:
     openEBSHelmChartName: "openebs/openebs"
     openEBSHelmRepoUrl: "https://openebs.github.io/openebs"
     openEBSProductNamespace: "openebs"
+    openEBSHelmChartVersion: "4.3.0-develop"
```

Comment:
I think you should make this dynamic and if possible directly read for openebs chart ?

Reply:
yes, we can make it dynamic going forward.